test/Interop/Cxx/operators/non-member-inline.swift

```diff
@@ -43,4 +43,22 @@ OperatorsTestSuite.test("slash") {
   expectEqual(1, result.value)
 }
 
+OperatorsTestSuite.test("less less (<<)") {
```

Comment:
Let's be consistent and either drop the parentheses in the test names or add them above as well.

Reply:
I just made this change myself.

Reply:
Thank you :)